src/stactools/landsat/constants.py

```diff
@@ -51,3 +51,5 @@ class Sensor(Enum):
         "thermal_gsd": 100,
     },
 }
+
+COORD_PRECISION = 6
```

Comment:
```suggestion
COORDINATE_PRECISION = 6
```

Nitpicky, but IMO no reason to abbreviate unless we already use "coord" as an abbreviation in this library.

Reply:
NP. Done.